apps/marginfi-v2-ui/src/components/common/Bridge/Bridge.tsx

```diff
@@ -45,41 +49,28 @@ const colors: MayanWidgetColors = {
 
 const rpcs = {
   solana: config.rpcEndpoint,
-  ethereum: process.env.NEXT_PUBLIC_ETHEREUM_RPC_ENDPOINT,
```

Comment:
@mohandesjiri does this mean we wont support these eth L2's with the removal of these RPCs?

Reply:
No, by removing the custom rpcs the widget will use the public rpcs